gdk/static/config_schema.json

```diff
@@ -64,21 +64,50 @@
                                     "build_system"
                                 ]
                             },
-                            "if": {
-                                "properties": {
-                                    "build_system": {
-                                        "const": "custom"
+                            "allOf": [
+                                {
+                                    "if": {
+                                        "properties": {
+                                            "build_system": {
+                                                "const": "zip"
+                                            }
+                                        }
+                                    },
+                                    "then": {
+                                        "properties": {
+                                            "options": {
+                                                "type": "object",
+                                                "description": "configuraiton options for the zip build system",
+                                                "required": ["exclude"],
```

Comment:
I'm not sure how the "options" for "zip" build system are gonna be extended yet. So, I don't think we can go with "required" here for exclude.

Reply:
Right now it is required given it is the only attribute. Moving forward might not be. This config might change a bit in order for us to support zipping multiple folders

Reply:
Making a field required to not required is safe (2 way door) if we want to change it later on. The other way around is not.